scripts/test-samples/bad-linebreaks-sample-before.md

```diff
@@ -59,6 +59,21 @@ let biscuits = "delicious";
   *     one   more       
   +     time  
 
+RPR: thanks especially to Roderigo for helping out to arrange that meeting. We can see you 
+in the room, Roderigo. Thank you. All right. So on to the usual items. So let’s start, 
+let’s see if we have approval for last -- actually, I don’t think we fully committed them yet, 
+so we should probably -- let’s return to that perhaps later in the meeting once we’ve got that. 
```

Comment:
This isn't quite the same, it has trailing whitespace.

```suggestion
so we should probably -- let’s return to that perhaps later in the meeting once we’ve got that.
```

Reply:
yes, the trailing whitespace is in the original notes!  but this is also the key difference in why this succeeds vs after the markdownlint fix.  will elaborite in the other PR